src/Cake.Common/Tools/TextTransform/TextTransformAliases.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.TextTransform
+{
+    /// <summary>
+    /// Contains functionality related to transforming templates
+    /// </summary>
+    [CakeAliasCategory("TextTransform")]
+    public static class TextTransformAliases
+    {
+        /// <summary>
+        /// Transform a text template.
+        /// </summary>
+        /// <example>
+        /// <code>
+        /// // Transform a .tt template.
+        /// var transform = File("./src/Cake/Transform.tt")
+        /// TransformTemplate(transform, new TextTransformSettings { OutputFile="./src/Cake/Transform.cs" });
```

Comment:
I had settings set to a nullable parameter.  I have since changed it to have a true overload.  For my learning, why was the nullable parameter not ideal?